### ISSUE TITLE
VideoCapture for file access

### ISSUE THREAD

**Issue by opencv-pushbot** [opencv/opencv#4551]
Transferred from http://code.opencv.org/issues/2970

```
|| Zdenek Kalal on 2013-04-17 09:04
|| Priority: Normal
|| Affected: 2.4.0 - 2.4.8
|| Category: highgui-images
|| Tracker: Bug
|| Difficulty: 
|| PR: 
|| Platform: Any / Any
```
## VideoCapture for file access

```
I use VideoCapture to access images in directory. The class works when the files are accessed sequentially. The following piece of code suppose to sequentially read images in directory but when the line vc.set(CV_CAP_PROP_POS_FRAMES, i) is used, it suddenly reads images from middle of the sequence.

@VideoCapture vc("data/%06d.jpg");
 int count = vc.get(CV_CAP_PROP_FRAME_COUNT);

 Mat img;
 for (int i = 0; i < count; i++)
 {
   // BUG ???
   vc.set(CV_CAP_PROP_POS_FRAMES, i); 
   // when commented the code works file

   int idx = vc.get(CV_CAP_PROP_POS_FRAMES);
   vc >> img;
   imshow("test",img);
   char ch = waitKey(0);
   if (ch == 27) break;
 }@
```
##### Evgeny Talanin on 2013-04-22 06:09

```
-   Assignee set to Vadim Pisarevsky
-   Category set to imgproc, video
```
##### Vadim Pisarevsky on 2015-04-28 12:02

```
-   Affected version changed from 2.4.0 - 2.4.4 to 2.4.0 - 2.4.8
-   Operating System set to Any
-   HW Platform set to Any
-   Category changed from imgproc, video to highgui-images
```

**Comment by bodepudiramakrishna** [comment]
I try to reproduce the bug and it is working fine in both cases(with vc.set and with out vc.set function) .

**Comment by bodepudiramakrishna** [comment]
I tried to reproduce the bug with the following setup with the functions   vc.set(CV_CAP_PROP_POS_FRAMES, i) commenting  and without commenting,but not able to reproduce . 
-   opencv version 3.1.0
-   Operating System Ubuntu 14.04

I hope this got fixed in the opencv-3.1.0. If the issue is fixed can this bug be closed?
